### PR TITLE
fix typo in hubspot spec example

### PR DESCRIPTION
**Pull request by jrhizor**

### CODE REVIEW

File: airbyte-integrations/connectors/source-hubspot-singer/Dockerfile [hunk 1]
```diff
@@ -8,7 +8,7 @@ ENV CODE_PATH="source_hubspot_singer"
 ENV AIRBYTE_IMPL_MODULE="source_hubspot_singer"
 ENV AIRBYTE_IMPL_PATH="SourceHubspotSinger"
 
-LABEL io.airbyte.version=0.1.5
+LABEL io.airbyte.version=0.1.6
```
Comment (sherifnada) [inline airbyte-integrations/connectors/source-hubspot-singer/Dockerfile:11]: bump in yml/json too and `/publish`?

**Review by sherifnada — APPROVED** [review]

**Comment by sherifnada** [comment]
@jrhizor you'll need to bump the JSON file as well

**Comment by sherifnada** [comment]
woops mb didn't see

**Comment by jrhizor** [comment]
Only reason I saw it was because it was in one of Artem's screenshots :-p

**Pull request merged**